conkit/core/contactmap.py

```diff
@@ -894,6 +909,39 @@ def remove_neighbors(self, min_distance=5, max_distance=sys.maxsize, inplace=Fal
                 contact_map.remove(tuple(contactid))
         return contact_map
 
+    def filter(self, threshold, filter_by='raw_score', inplace=False):
+        """Filter out contacts below selected threshold
+
+        Parameters
+        ----------
+        threshold : int
+           Threshold to be applied in the filter
+        filter_by : str
+            Contact attribute to be used in the filter.
+        inplace : bool, optional
```

Comment:
Not sure if I should check that `filter_by` is a valid `conkit.core.contact.Contact` attribute